spec/system/onboardings/user_completes_onboarding_spec.rb

```diff
@@ -28,26 +30,19 @@
   end
 
   context "when the user has seen onboarding" do
-    before do
-      user.update(saw_onboarding: true)
```

Comment:
Rather than updating the user's `saw_onboarding` attribute, I just created a new one for this context.